modules/OFStateManager/module/src/gentable_handlers.c

```diff
@@ -185,45 +214,49 @@ indigo_core_gentable_unregister(indigo_core_gentable_t *gentable)
 
 /* OpenFlow message handlers */
 
-void
-ind_core_bsn_gentable_entry_add_handler(
+/**
+ * gentable add/modify resume functon
+ * Drivers will call this resume function if the async operation reaches
+ * the final status
+ * If it is not an async operation, the indigo will call it.
+ * (The existing non-async operation deson't need to modify the codes.)
```

Comment:
"does not"
do you mean "Existing non-async drivers do not need modification" ?  if so, let's just write that.

Reply:
Rewrite the comments